### DISCUSSION TITLE
Une fin ! Et un recommencement

### DISCUSSION THREAD

**Discussion by Procion06** [Crownicles/Crownicles#1371] [Nouvelles idées]
Bonjour, 

Comme tout bon jeu a toujours une bonne fin je propose de rajouter un event secret qui se débloque sous certains prérequis, où on retrouve enfin cette putain de Princesse !
Pour les prérequis j'ai 2 propositions à vous faire :

1/ avoir atteint le level 100 (je pense que ça correspond à 1 an de jeu mais vous êtes mieux placés que moi pour le dire)
ou
2/ réaliser une "mini-campagne" finale composée de 5 quêtes plutôt longues (que je me propose de créer au besoin)
   avoir atteint le level 80

Qui dit bonne fin dit bon loot :

1 item légendaire (ou mythique) au hasard parmi 5 items créés et dédiés pour cette fin

Ce serait le cadeau de la Princesse ou du Duc en remerciement
Une fois ce dernier event enfin accompli, les levels du joueur sont réinitialisés et il repart du level 0, mais avec un bel item en plus dans son équipement

Dites-moi ce que vous en pensez, si l'idée vous plaît (avec ou sans améliorations) ou si c'est de la merde

**Comment by Easygamer01s** [comment]
Perso je trouve l'idée brillante ! le Shadowmourne (wotlk) de Draftbot =D
Il faudrait que les défis soient d'une difficulté hors du commun et demandant une vraie action de la part du joueur.

Par contre je trouverais ça dommage que le level soit remis à zéro, sauf si vraiment cette arme vaut aussi le coup en matière de stat.
Level 95 -> 2,5 ans je dirais =D
